application/cms/filters.py

```diff
@@ -58,3 +58,9 @@ def yesno(state):
         return "no"
 
     return state
+
+
+def models_to_dicts(items):
```

Comment:
This is only used in the measure page template, and I can't see it being used elsewhere much at all, so maybe could just pass the Dimension dicts in to the template directly from the view without needing this new filter at all?

I'm not strong on this either way I guess, but this seems a pretty odd filter to have.

Reply:
I'd rather that our 'standard' is for templates to have the full model instances available at all times, as that feels like a more reliable and natural interface. We only need this filter when we want to expose the list of model instances to JavaScript - which is arguably a pretty odd thing to do in the first place.

Do you think it's better the way you suggest?